### PR TITLE
Removed unnecessary authenticate popup block from print layout.

### PR DESCRIPTION
**Pull request by pratiksangani**
### Description (*)

Removed unnecessary authenticate popup block from print layout as an addition to https://github.com/magento/magento2/pull/30646 PR.

### Related Pull Requests

https://github.com/magento/magento2/pull/30646

### Fixed Issues (if relevant)

1. https://github.com/magento/magento2/issues/29208

### Manual testing scenarios (*)

N/A

### Questions or comments

N/A

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Functional Tests CE

**Comment by ihor-sviziev** [comment]
@slavvka @sidolov could you provide some info why functional tests are failing, while no failures in test results?

**Comment by sidolov** [comment]
@ihor-sviziev I don't see errors that could cause the failure, I have restarted the build

**Comment by ihor-sviziev** [comment]
@sidolov now the same

File: app/code/Magento/Theme/view/frontend/layout/print.xml [hunk 1]
```diff
@@ -16,5 +16,6 @@
         <referenceContainer name="before.body.end">
             <block class="Magento\Framework\View\Element\Template" name="sales.order.print.js" template="Magento_Theme::html/print.phtml" />
         </referenceContainer>
+        <referenceBlock name="authentication-popup" remove="true"/>
```
Comment (ihor-sviziev) [inline app/code/Magento/Theme/view/frontend/layout/print.xml]: I see this block is defined in the Magento/Customer module. Could you move your change to this module as well?
https://github.com/magento/magento2/blob/a360c06684322189609dd543d09bc31d8d2bdca0/app/code/Magento/Customer/view/frontend/layout/default.xml#L26

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Theme/view/frontend/layout/print.xml (hunk 1 above)
Comment (pratiksangani) [reply app/code/Magento/Theme/view/frontend/layout/print.xml]: Sure @ihor-sviziev

Comment (pratiksangani) [reply app/code/Magento/Theme/view/frontend/layout/print.xml]: @ihor-sviziev Your suggested changes were done. Will you please review & proceed with my PR?

Comment (ihor-sviziev) [reply app/code/Magento/Theme/view/frontend/layout/print.xml]: Hi @pratiksangani,
Now we basically removing this block in two modules that is not correct. We should remove it only in customer module, there should not be any changes in “theme” module

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

**Comment by pratiksangani** [comment]
@ihor-sviziev You're correct. I've reverted back changes from `Magento_Theme` module. Please verify it now.

**Comment by ihor-sviziev** [comment]
Now it looks good.
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run WebAPI Tests, Functional Tests B2B, Functional Tests EE

**Review by ihor-sviziev — APPROVED** [review]

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8427](https://jira.corp.magento.com/browse/ENGCOM-8427) has been created to process this Pull Request

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Comment by engcom-Dash** [comment]
Hello @pratiksangani,

Thank you for your contributions!

I tried to replicate the issue but was unable to do so because it was fixed in an internal Jira ticket. The related internal Jira ticket MC-36218 was closed as Fixed.
image


Please see https://github.com/magento/magento2/issues/29208#issuecomment-790594409

Hence, closing this PR.

Thanks.

**Pull request closed without merging**